zerver/lib/actions.py

```diff
@@ -1052,6 +1063,13 @@ def should_send(setting: str, row: Dict[str, Any]) -> bool:
             if should_send('email_notifications', row)
         }
 
+        for row in contributor_rows:
+            row['is_muted'] = row['user_profile_id'] in user_ids_muting_stream
```

Comment:
Yeah, in fact, I think if you just delete this line and the query just above, the right thing will happen.